content/facility.yaml

```
@@ -1,168 +1,153 @@
 Prototyping Tools:
```

It seems Prusa scrubbed their official product page for the MK3S+ printers, so I have to use an archived link for now.
```suggestion
Prototyping Tools:
  - name: Prusa i3 MK3S+
    img: prusa-mk3s
    url: https://web.archive.org/web/20230223035146/https://www.prusa3d.com/category/original-prusa-i3-mk3s

```